core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -4739,6 +4739,7 @@ groups:
     - branch-ci-openshift-release-master-release-controller-annotate
     - branch-ci-openshift-release-master-vsphere-apply
     - openshift-release-master-config-bootstrapper
+    - periodic-github-ldap-user-group-creator
```

Comment:
Shouldn't be needed after https://github.com/openshift/release/pull/22198

Reply:
Removed.